test/server/contentBase-option.test.js

```diff
@@ -301,6 +301,72 @@ describe('contentBase option', () => {
         done();
       }
     });
+    it('Should not throw exception (local path with lower case first character)', (done) => {
+      try {
+        // eslint-disable-next-line no-unused-vars
+        server = new Promise((resolve, reject) => {
+          testServer.start(config, {
+            contentBase:
+              contentBasePublic.charAt(0).toLowerCase() +
+              contentBasePublic.substring(1),
+            watchContentBase: true,
+            port: 2222,
```

Comment:
Don't use the port number directly.

Reply:
if I use `port` it'll cause `listen EADDRINUSE: address already in use 127.0.0.1:8107` error and cause the tests to hang, can I use `port+1`,`port+2` or add new values in `ports-map` ?

Reply:
Don't add the port number.

<details>

```diff
diff --git a/test/server/contentBase-option.test.js b/test/server/contentBase-option.test.js
index fe120e5..bcde1e6 100644
--- a/test/server/contentBase-option.test.js
+++ b/test/server/contentBase-option.test.js
@@ -269,10 +269,10 @@ describe('contentBase option', () => {
 
   describe('testing single & multiple external paths', () => {
     afterEach((done) => {
-      testServer.close(() => {
-        done();
-      });
+      console.log('aa');
+      testServer.close(done);
     });
+
     it('Should throw exception (string)', (done) => {
       try {
         // eslint-disable-next-line no-unused-vars
@@ -302,71 +302,40 @@ describe('contentBase option', () => {
       }
     });
     it('Should not throw exception (local path with lower case first character)', (done) => {
-      try {
-        // eslint-disable-next-line no-unused-vars
-        server = new Promise((resolve, reject) => {
-          testServer.start(config, {
-            contentBase:
-              contentBasePublic.charAt(0).toLowerCase() +
-              contentBasePublic.substring(1),
-            watchContentBase: true,
-            port: 2222,
-          });
-          resolve(testServer);
-        });
-
-        server.then((testServer) => {
-          testServer.close(() => {
-            done();
-          });
-        });
-      } catch (e) {
-        expect(true).toBe(false);
-      }
+      testServer.start(
+        config,
+        {
+          contentBase:
+            contentBasePublic.charAt(0).toLowerCase() +
+            contentBasePublic.substring(1),
+          watchContentBase: true,
+          port,
+        },
+        done
+      );
     });
     it("Should not throw exception (local path with lower case first character & has '-')", (done) => {
-      try {
-        // eslint-disable-next-line no-unused-vars
-        server = new Promise((resolve, reject) => {
-          testServer.start(config, {
-            contentBase: 'c:\\absolute\\path\\to\\content-base',
-            watchContentBase: true,
-            port: 2223,
-          });
-          resolve(testServer);
-        });
-
-        server.then((testServer) => {
-          testServer.close(() => {
-            done();
-          });
-        });
-      } catch (e) {
-        expect(true).toBe(false);
-      }
+      testServer.start(
+        config,
+        {
+          contentBase: 'c:\\absolute\\path\\to\\content-base',
+          watchContentBase: true,
+          port,
+        },
+        done
+      );
     });
     it("Should not throw exception (local path with upper case first character & has '-')", (done) => {
-      try {
-        // eslint-disable-next-line no-unused-vars
-        server = new Promise((resolve, reject) => {
-          testServer.start(config, {
-            contentBase: 'C:\\absolute\\path\\to\\content-base',
-            watchContentBase: true,
-            port: 2224,
-          });
-          resolve(testServer);
-        });
-
-        server.then((testServer) => {
-          testServer.close(() => {
-            done();
-          });
-        });
-      } catch (e) {
-        expect(true).toBe(false);
-      }
+      testServer.start(
+        config,
+        {
+          contentBase: 'C:\\absolute\\path\\to\\content-base',
+          watchContentBase: true,
+          port,
+        },
+        done
+      );
     });
-
     it('Should throw exception (array)', (done) => {
       try {
         // eslint-disable-next-line no-unused-vars
```

</details>

You can patch this code.